setup.py

```diff
@@ -55,6 +55,7 @@
         "google-api-core >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
         "pandas >= 1.0.0, <3.0.0",
         "pandas >= 1.0.0, <2.0.0; python_version<'3.8'",
+        "pyarrow >= 15.0.0, <16.0.0; python_version>='3.8'",  # Required by Pandas #237
```

Comment:
Include the full github issue link "https://github.com/googleapis/python-documentai-toolbox/issues/237" instead.